libs/hibiscus-supabase-client/src/supabase-client.tsx

```diff
@@ -25,9 +25,24 @@ export class HibiscusSupabaseClient {
   private client: SupabaseClient;
 
   constructor() {
+    const apiUrl = getEnv().Hibiscus.Supabase.apiUrl;
+    const anonKey = getEnv().Hibiscus.Supabase.anonKey;
+
+    if (!apiUrl) {
+      console.error(
+        'Supabase API URL not provided. Placeholder values will be used. The app will not be able to access Supabase.'
+      );
+    }
+
+    if (!anonKey) {
+      console.error(
+        'Supabase anon key not provided. Placeholder values will be used. The app will not be able to access Supabase.'
+      );
+    }
+
     this.client = createClient(
-      process.env.NEXT_PUBLIC_HIBISCUS_SUPABASE_API_URL,
-      process.env.NEXT_PUBLIC_HIBISCUS_SUPABASE_ANON_KEY
+      apiUrl ?? 'http://placeholder',
+      anonKey ?? 'placeholder'
```

Comment:
@snokpok It still wouldn't build on CI even if I moved the context into dashboard app, so I did this to kind of work around the the CI. Is this ok?